File: solved.ac/code/13458.py
import sys
input = sys.stdin.readline
n = int(input())
a = list(map(int, input().split()))
b,c = map( int, input().split())

ans = len(a)
for i in range(len(a)):
  a[i] -= b
for i in range(len(a)):
  if a[i] <= 0:
    continue
  else:
    if a[i] % c == 0:
      ans += a[i] // c
    else:
      ans += a[i] // c  + 1 
    # while a[i] > 0 :
    #   a[i] -= c
    #   ans += 1
print(ans)

File: solved.ac/javascript/13458.js
const fs = require("fs")
const input = fs.readFileSync("./dev/stdin").toString().trim().split("\n")
function solution(input) {
  const n = input.shift()
  console.log(input)
  const [main, sub] = input[1].split(" ").map((e) => Number(e))
  const exam = input[0].split(" ").map((e) => Number(e) - main)
  let answer = Number(n)
  exam.forEach((e) => {
    if (e > 0) {
      answer += Math.ceil(e / sub)
    }
  })
  console.log(answer)
}
solution(input)

//   exam.forEach((e) => {
//     if (e > 0) {
//       answer += Math.ceil(e / sub)
//     }
//   })
//   console.log(answer)
// }

// solution(input)
